cibuild.sh

```diff
@@ -118,6 +123,7 @@ function xtensa-esp32-gcc-toolchain {
     rm xtensa-esp32-elf-gcc8_2_0-esp32-2019r1-rc2-linux-amd64.tar
   fi
   xtensa-esp32-elf-gcc --version
+  pip install esptool
```

Comment:
can we check some file doesn't exist before install esptool like we check $prebuilt/xtensa-esp32-elf/bin/xtensa-esp32-elf-gcc?

Reply:
`pip` performs this check and will skip downloading if it already exists so I think we are good.